internal/app/api/services/user/user.go

```diff
@@ -145,3 +147,48 @@ func (s *Service) UpdateUserByID(ctx context.Context, user types.User) error {
 	s.logger.Infof("updated user is completed ")
 	return err
 }
+
+// Get list users by page
+func (s *Service) GetListUsers(ctx context.Context, page, size string) (*types.GetListUsersResponse, error) {
+
+	var pagingNSorting types.PagingNSorting
+	err := pagingNSorting.Init(page, size)
+
+	if err != nil {
+		s.logger.Errorf("Failed url parameters when get list users", err)
+		return nil, err
```

Comment:
Create a new custom error message then return, the err from the repo is just to log